mesoscope/cellpack
# Feature/store runs in time

Problem
=======
closes #36 

Solution
========
created a mapping from seed number to results to store all the objects in one simularium file 
with @mogres @kvegesna @rugeli 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to not work as expected)

Steps to Verify:
----------------
1. change the parallel setting to false in the config file 
2. ` pack -r 'cellpack/examples/recipes/v2/one_sphere.json' -c '/cellpack/cellpack/tests/packing-configs/test_parallel_config.json'`
3. should open the _all file in simularium with multiple packings
